lib/voxpupuli/acceptance/spec_helper_acceptance.rb

```diff
@@ -21,9 +21,9 @@ def configure_beaker(modules: :metadata, &block)
       unless %w[none preinstalled].include?(collection)
```

Comment:
Late realization: the `collection2packagename` doesn't deal with these. I've submitted https://github.com/voxpupuli/beaker_puppet_helpers/pull/62 to deal with `none` in the helpers. Should `preinstalled` be dealt with by not installing any package at all?